examples/simultaneous-cursors/README.md

```diff
@@ -0,0 +1,61 @@
+# Yorkie Simultaneous-Cursors Example
+
+<p>
+    <a href="https://yorkie.dev/yorkie-js-sdk/examples/vanilla-quill/" target="_blank">
```

Comment:
Also you need to change URL to `https://yorkie.dev/yorkie-js-sdk/examples/simultaneous-cursors/`